src/User.php

```diff
@@ -23,6 +23,8 @@ class User implements UserInterface
 	public $title;
 	public $forename;
 	public $surname;
+	public $description;
+	public $jobTitle;
```

Comment:
Looking at this module as a whole, these fields don't seem to fit in. This class is really light, holding only essential information (it doesn't even have address). These two fields - especially jobTitle ` don't feel integral enough to what a User is to have here. It begins to change what this class actually represents, idk if that's cool.

(Consider someone using cog and cog-user without the context of mothership).